src/Hl7.Fhir.Specification/Validation/XmlValidationExtensions.cs

```diff
@@ -81,33 +79,33 @@ internal static OperationOutcome ValidatedParseXml(this Validator me, XmlReader
                 if (me.Settings.EnableXsdValidation)
                 {
                     var doc = XDocument.Load(instance, LoadOptions.SetLineInfo);
-                    result.Add(me.ValidateXml(doc));
+                    result.Add(validateXml(doc, xsdSchemas));
                     instance = doc.CreateReader();
                 }
 
                 poco = (Resource)(new FhirXmlParser()).Parse(instance, typeof(Resource));
             }
             catch (Exception e)
             {
-                result.AddIssue($"Parsing of Xml into a FHIR Poco failed: {e.Message}", Issue.XSD_CONTENT_POCO_PARSING_FAILED, (string)null);
+                result.AddIssue($"Parsing of Xml into a FHIR Poco failed: {e.Message}", Issue.XSD_CONTENT_POCO_PARSING_FAILED, default(string));
                 poco = null;
             }
 
             return result;
-        }
 
+            static OperationOutcome validateXml(XDocument instance, XmlSchemaSet xsdSchemas)
```

Comment:
missing here an access modifiers (internal probably)

Reply:
Haha, took me a while to notice that this is a nested function - so no access modifiers possible!